src/python_be.cc

```diff
@@ -387,10 +387,21 @@ ModelInstanceState::SaveRequestsToSharedMemory(
     auto err = TRITONBACKEND_RequestTrace(request, &triton_trace);
     if (err != nullptr) {
       triton_trace = nullptr;
+      LOG_MESSAGE(TRITONSERVER_LOG_ERROR, TRITONSERVER_ErrorMessage(err));
       TRITONSERVER_ErrorDelete(err);
     }
+    const char* val = nullptr;
+    if (triton_trace != nullptr) {
+      err = TRITONSERVER_InferenceTraceContext(triton_trace, &val);
+      if (err != nullptr) {
+        LOG_MESSAGE(TRITONSERVER_LOG_ERROR, TRITONSERVER_ErrorMessage(err));
+        TRITONSERVER_ErrorDelete(err);
+      }
```

Comment:
I think `LOG_IF_ERROR` does this, something like:
```suggestion
      LOG_IF_ERROR(TRITONSERVER_InferenceTraceContext(triton_trace, &val), "failed to get trace context");
```